templates/maldi-pipeline.ipynb

```diff
@@ -4,14 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<p>base_dir: specify file type extenstion (.d file or .imzml/.ibd?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/maldi-tools/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>@goterm just pushed new changes, do they address this?</p>